app/services/EventReportService.scala

```diff
@@ -175,13 +175,13 @@ class EventReportService @Inject()(eventReportConnector: EventReportConnector,
               logger.info(s"When data found in repo and event data changed is ${isDataChanged(oldData, newData)}")
               Future.successful(isDataChanged(oldData, newData))
             case (None, Some(_)) =>
-              logger.warn("New answers are available while old ones are not")
+              logger.info("New answers are available while old ones are not")
               Future.successful(true)
             case (Some(_), None) =>
-              logger.warn("Old answers are available while new ones are not")
+              logger.info("Old answers are available while new ones are not")
               Future.successful(true)
             case (None, None) =>
-              logger.warn("Neither user answers are there")
+              logger.info("Neither user answers are there")
```

Comment:
I think those should stay as warnings. Might indicate a problem. I don't know when this is possible.